src/components/DateTimePicker/index.jsx

```diff
@@ -31,38 +37,54 @@ const DateTimePicker = ({
   datePickerProps,
   timePickerProps,
 }) => {
-  const [open, setOpen] = React.useState(false);
-  const [time, setTime] = useState(value);
-
+  const [open, setOpen] = useState(datePickerProps?.open);
+  const [date, setDate] = useState();
+  const [time, setTime] = useState();
+  const [changedField, setChangedField] = useState();
```

Comment:
Can we add an eslint rule to catch state declarations which doesn't have initial value? I think being explicit during state setting is better than assuming or reading the whole component's code to figure out what value a state should be dealing with. Here I think setting to `null` would be more explicit as it's an intentional empty value. 

Another example: The `changeField` state when I glanced through PR changes seems to be dealing with string values - if that's the case then we ought to set the initial value of this state with an empty string.

Refer:
- We had a section in React best practices book - that book itself is 404 right now. :/

What are your thoughts?

Reply:
I can add explicit initial values. About the rule, well one of the reason people use Typescript.

Reply:
Yes we can add an eslint rule for this. I will create an issue for this.

Reply:
Issue: https://github.com/bigbinary/eslint-plugin-neeto/issues/561